modules/raft/src/main/java/org/apache/ignite/raft/jraft/storage/impl/RocksDbSharedLogStorage.java

```diff
@@ -123,8 +121,11 @@ private static class EmptyWriteContext implements WriteContext {
     /** Write lock. */
     private final Lock writeLock = this.readWriteLock.writeLock();
```

Comment:
It seems that `readLock` and `writeLock` are not optimal names here, because they are not about reading and writing (both for read/write operations in this class, readLock is taken), but for 'managing the service' (start/stop/reset) and 'using' it. How about renaming them to `manageLock` and `useLock`, or `exclusiveLock` and `nonExclusiveLock`?

Just a suggestion :)